lib/taskana-core/src/test/java/pro/taskana/impl/integration/TaskServiceImplIntExplicitTest.java

```diff
@@ -398,6 +398,7 @@ void shouldNotTransferAnyTask() throws SQLException {
       userName = "User",
       groupNames = {"businessadmin"})
   @Test
+  @SuppressWarnings("checkstyle:VariableDeclarationUsageDistance")
```

Comment:
why not making them final?
We have checkstyle for a reason :P

Reply:
good catch! done